src/client/commands/worker.rs

```diff
@@ -47,8 +48,23 @@ pub async fn get_worker_info(
 
 pub async fn stop_worker(
     connection: &mut ClientConnection,
-    worker_id: WorkerId,
+    selector: WorkerSelector,
 ) -> crate::Result<()> {
-    let message = FromClientMessage::StopWorker(StopWorkerMessage { worker_id });
-    rpc_call!(connection, message, ToClientMessage::StopWorkerResponse).await
+    let message = FromClientMessage::StopWorker(StopWorkerMessage { selector });
+    let mut responses =
+        rpc_call!(connection, message, ToClientMessage::StopWorkerResponse(r) => r).await?;
+
+    responses.sort_unstable_by_key(|x| x.0);
+    for (id, response) in responses {
+        match response {
+            StopWorkerResponse::Failed(e) => {
+                println!("Worker: {}, failed to stop, error: {}", id, e.to_string())
```

Comment:
This should probably be returned as an error? But we would have to combine all of the error messages into a single `Result` to avoid losing data. Or at least use `eprintln` so that it is printed to stderr instead of stdout.

Reply:
@spirali Maybe just logging to error would be enough here? To avoid complex combination of multiple errors into a single Result.